reference/shared/office.context.auth.getAccessTokenAsync.md

```diff
@@ -48,19 +48,21 @@ getAccessTokenAsync([Options,] callback);
 
 ##Examples
 
-```
-Office.context.auth.getAccessTokenAsync(handleAuthToken);
-
-...
+	Office.context.auth.getAccessTokenAsync(function(result) {
+	    if (result.status === "succeeded") {
+		var token = result.value.accessToken;
```

Comment:
This makes the sample more complete, but you still want many more examples, of much greater detail